index.html

```diff
@@ -1238,6 +1238,369 @@ <h2>Evidence</h2>
         </p>
 
       </section>
+<section>
+
+      <h1>Subject Not Equal To Holder</h1>
+
+      <p>
+This section describes how the data model supports use cases in which the holder
+ of the credential is not the subject of the credential.
+      </p>
+
+      <p>
+The following diagram depicts those instances when the subject is not equal to 
+the holder.
+     </p>
+
+      <figure>
+        <img style="margin: auto; display: block;" width="75%" src="diagrams/subject-ne-holder.svg">
+        <figcaption style="text-align: center;">
+Subject Holder Relationships in Verifiable Credentials.
+        </figcaption>
+      </figure>
+
+     <p>
+The following sections describe how each of these instances are handled in 
+the data model
+     </p>
+
+
+    <section>
+        <h2>Credential Uniquely Identifies Subject</h2>
+
+     <p>
+In this case, the claim may contain multiple properties that each provide 
+an aspect of the identity of the subject, and which together unambiguously identify
+the subject. In this case, it does not matter who the holder is, since the verifier
+can unambiguously identify the subject of the verifiable credential.
+   </p>
+
+        <pre class="example nohighlight" title="An example of a credential 
+that uniquely identifies a subject">
+
+{
+	"id": "http://dmv.example.gov/credentials/3732",
+	"type": ["Credential", "ProofOfAgeCredential"],
+	"issuer": "https://dmv.example.gov/issuers/14",
+	"issued": "2010-01-01",
+	"claim": {
+		"id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+		"name": "Mr John Doe",
+		"address": "10 Some Street, Anytown, ThisLocal, Country X",
+		"dateOfBirth": "2001-01-01",
+		"gender": "male",
+		"placeOfBirth": "Anytown, ThisLocal, Country X",
+		"nationality": "X",
+	},
+	"proof": { ...
+	}
+}
+        </pre>
+
+
+   </section>
+
+    <section>
+       <h2>Subject Delegates to Holder</h2>
+
+     <p>
+This is supported in the data model in two different ways: through the use of either recursion or verifiable presentations. Both methods involve the subject issuing a new verifiable credential to the delegated holder.
+    </p>
+
+    <p>
+This process of recursion could continue without restriction. Therefore 
+a new property “delegatable”, is added to the data model.
+
+     </p>
+
+   <section>
+
+   <h3> Delegatable Property </h3>
```

Comment:
This is stomping on what we expect OCAPs to be used for... don't know how I feel about this other than it makes me uneasy. Can you at least put a <p class="issue"> beside it stating that the WG has not determined if this is the proper approach yet? I don't want this to keep this PR from getting into the spec.

Reply:
We have had long discussions about OCAPs vs VCs and it is clear that there are separate use cases and roles for both of these.
Role delegation in organisations is very common - so common in fact that most organisations could not function without the ability to delegate roles. This is why the example is the delegation of a role.
RBAC is also very common in organisations, and role delegation is needed in this.

Reply:
> Role delegation in organisations is very common

I'd argue that role delegation in organizations, while common, is executed in very different ways... some of them, dangerously. I'd also argue that we need to be crystal clear about what "role delegation" means... because in may cases, it means "I'm granting permission for person X to do action Y for time period Z." ... but in reality, what usually happens is "I'm enabling person X to be me without restriction".

... and that's what it looks like you're modeling here, which is why I'm objecting to the data model enabling that scenario. It's the text book definition of ambient authority - https://en.wikipedia.org/wiki/Ambient_authority